openedx/core/lib/xblock_builtin/xblock_discussion_course/static/js/discussion_course.js

```diff
@@ -1,6 +1,16 @@
-/* globals DiscussionUtil */
+/* globals window, $$course_id, DiscussionUtil */
 function DiscussionCourseBlock(runtime, element) {
     'use strict';
+    var has_push = window.history && window.history.pushState ? true : false;
```

Comment:
@pomegranited Can you rename this to `hasPush` (or `hasPushState`)? I used variable names with underscores in JS recently, and Jenkins reported quality failures because of it. (We won't submit this code upstream, but it's probably still worth to try and converge on a consistent naming scheme.)